kinto/views/buckets.py

```diff
@@ -1,7 +1,7 @@
 from kinto.core import resource
 from kinto.core.events import ResourceChanged, ACTIONS
 from pyramid.events import subscriber
-
+from kinto.authorization import RouteFactory
```

Comment:
> we shouldn't have multiple RouteFactory for a given project

That's not a problem if their code is small. Having tests like `if self.resource_name == 'bucket'` bothers me.

Look nothing says here that we shouldn't have one per route:
http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/urldispatch.html#route-factories